src/components/PrivacyModal/PrivacyModal.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState, memo, useCallback } from 'react'
+import { useLocation, useNavigate } from 'react-router-dom'
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+  display: 'flex'
+}
+
+/**
+ * @prop usePrivacyModal - Hook used for the Modal logic upon clicking a button.
+ * @interface
+ */
+
+export type PrivacyModalProps = {
+  usePrivacyModal?: () => PrivacyModalHookReturn
+}
+
+/**
+ * PrivacyModal component.
+ *
+ * @param props - Props containing the logic to accept and decline the privacy policy cookie.
+ *
+ * @remarks
+ * PrivacyModal shows a Modal for the user to read and accept the PrivacyPolicy and other.
+ * A Formcontrol label can be added for other documents that need to be read by the user.
+ * The PrivacyModal gets shown upon entering HASKI.
+ * Gets rendered in App.
+ *
+ * @category Components
+ */
+
+const PrivacyModal = ({ usePrivacyModal = _usePrivacyModal }: PrivacyModalProps) => {
+  const { t } = useTranslation()
+  const navigate = useNavigate()
+  const [open, setOpen] = useState(true)
+  const [checked, setChecked] = useState(false)
+  const { privacyPolicyCookie, handleAccept } = usePrivacyModal()
+  const currentLocation = useLocation()
+
+  //Disable backdropClick so the Modal only closes via the buttons
+  const handleClose = useCallback((_: React.MouseEvent<HTMLElement>, reason: string) => {
+    if (reason && reason == 'backdropClick') return
+  }, [])
+
+  //setting the check so the button gets enabled
+  const handleChecked = useCallback(
+    (event: React.MouseEvent<HTMLElement>) => {
+      setChecked((event.target as HTMLInputElement).checked)
+    },
+    [setChecked]
+  )
+
+  //close the modal and set the cookie in the hook
+  const handleModal = useCallback(
+    (cookieAccepted: boolean) => {
+      handleAccept(cookieAccepted)
+      setOpen(false)
+    },
+    [handleAccept, setOpen]
+  )
+
+  //load nothing if cookie is set or privacypolicy page is loaded
+  if (privacyPolicyCookie || currentLocation.pathname.includes('privacypolicy')) return null
+
+  return (
+    <Modal
+      aria-labelledby="privacy-modal"
+      aria-describedby="makes-the-user-accept-the-privacypolicy-or-not"
+      open={open}
+      onClose={handleClose}
+      closeAfterTransition>
+      <Box sx={style}>
+        <Typography id="transition-modal-title" variant="h4" component="h2">
+          {t('components.PrivacyModal.termsOfService')}
+          <Typography id="modal-text" variant="subtitle1" component="h2">
+            {t('components.PrivacyModal.afterReading')}
+            <FormGroup>
+              <FormControlLabel
+                onClick={handleChecked}
+                control={<Checkbox />}
+                label={
+                  <>
+                    {t('components.PrivacyModal.readPrivacypolicy')}&nbsp;
```

Comment:
Der non-breaking space hier hat mich etwas verwirrt.
Wäre es auch möglich das Leerzeichen in den Translation Files hinzuzufügen? Insofern es nicht irgendein gewolltes Behaviour zerstört.